extensions/sd_api_pictures/script.py

```diff
@@ -96,6 +105,14 @@ def get_SD_pictures(description):
         image_bytes = buffered.getvalue()
         img_str = "data:image/jpeg;base64," + base64.b64encode(image_bytes).decode()
         visible_result = visible_result + f'<img src="{img_str}" alt="{description}">\n'
```

Comment:
Suggestion: if `save_img` is on, skip this shrink and embed step, and instead embed `<img src="/file/extensions/etcetc.png">`. Makes for much cleaner+shorter data, and allows full original res to be used.

Reply:
Also, please make image size configurable, allowing to not resize it at all.

Reply:
> Also, please make image size configurable, allowing to not resize it at all.

That's not defined here, the 300x300 viewport is set in the main ooba css. I could probably override it, but the resizing is done just to match the viewport